Libraries/Text/Text/RCTTextView.m

```diff
@@ -423,34 +459,28 @@ - (BOOL)becomeFirstResponder
 
 - (BOOL)resignFirstResponder
 {
-  if (![super resignFirstResponder]) {
+  // Don't relinquish first responder while selecting text.
+  if (_selectable && NSRunLoop.currentRunLoop.currentMode == NSEventTrackingRunLoopMode) {
     return NO;
   }
 
-  // If we've lost focus, notify listeners
-  [_eventDispatcher sendEvent:[RCTFocusChangeEvent blurEventWithReactTag:self.reactTag]];
-
-  return YES;
+  return [super resignFirstResponder];
 }
 
-#endif // ]TODO(macOS GH#774)
-
-- (BOOL)canBecomeFirstResponder
+- (BOOL)canBecomeKeyView
 {
+  // RCTText should not get any keyboard focus unless its `selectable` prop is true
   return _selectable;
```

Comment:
In our fork we override resignFirstResponder and canBecomeKeyView on the NSTextView subclass, not on RCTTextView. Do we need to move these there? Also we always return NO from canBecomeKeyView.

Reply:
Good call. RCTTextView can be the key view but I do think we want the actual NSTextView to be selected here.
After investigating, I'm not sure the `resignFirstResponder` fix is doing anything here.

See comment under focus view for why we should use `_selectable` to determine `canBecomeKeyView` value.